stylesheets/_component.navigation.scss

```diff
@@ -360,6 +366,115 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
     }
 }
 
+.nav-tertiary {
```

Comment:
Is that better than abstracting out the common styles into a multiple selector (thinking Hunger Games here...)